engines/dfc_provider/app/services/enterprise_importer.rb

```diff
@@ -1,18 +1,44 @@
 # frozen_string_literal: true
 
 class EnterpriseImporter
-  def import(owner, dfc_enterprise)
-    address = dfc_enterprise.localizations.first
+  def initialize(owner, dfc_enterprise)
+    @owner = owner
+    @dfc_enterprise = dfc_enterprise
+  end
+
+  def import
```

Comment:
Should we have specs for this ? I don't think the update scenario is tested anywhere ?

Reply:
It's covered by the dfc_importer_spec.